src/virtual-repeat.js

```diff
@@ -411,7 +411,14 @@ export class VirtualRepeat extends AbstractRepeater {
     let firstViewElement = this.view(0).lastChild;
     this.itemHeight = calcOuterHeight(firstViewElement);
     if (this.itemHeight <= 0) {
-      throw new Error('Could not calculate item height');
+      this._sizeInterval = setInterval(()=>{
```

Comment:
`this._sizeInterval` might need to be terminated in the detached callback as well. In case the item height never gets greater than 0.